napari/utils/io.py

```diff
@@ -1,16 +1,85 @@
 import os
+import re
 
 from glob import glob
 from pathlib import Path
 
 import numpy as np
-from skimage import io
-from skimage.io.collection import alphanumeric_key
 
 from dask import delayed
 from dask import array as da
 
 
+def imsave(filename: str, data: np.ndarray):
+    """custom imaplementation of imread to avoid skimage dependecy"""
```

Comment:
```suggestion
    """Custom implementation of imsave to avoid skimage dependency.

    Parameters
    ----------
    filename : string
        The path to write the file to.
    data : np.ndarray
        The image data.
    """
```